download_test_data.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -eu
+
+TEST_DATA_URL=${TEST_DATA_URL:-https://big-tank.app.tu-dortmund.de/lst-testdata/}
+TEST_DATA_USER=${TEST_DATA_USER:-""}
+TEST_DATA_PASSWORD=${TEST_DATA_PASSWORD:-""}
+
+
+if [ -z "$TEST_DATA_USER" ]; then
+	echo -n "Username: "
+	read TEST_DATA_USER
+	echo
+fi
+
+if [ -z "$TEST_DATA_PASSWORD" ]; then
+	echo -n "Password: "
+	read -s TEST_DATA_PASSWORD
+	echo
+fi
+
+
+wget \
```

Comment:
add `-nc` to avoid human users re-downloading data when running `pytest` on their laptop?

Reply:
don't you download the data once first and that's it? Does `pytest` try to download the data everytime you run it locally?

Reply:
Yes, good option

Reply:
No, pytest does not do any downloading. But we should make sure that people can simply rerun the command when we update data on the server.

Reply:
But I think better to go with `-N` instead of `-nc`

Reply:
ok, I see

Reply:
but maybe new data should be checked for everytime pytest is run?
Imagine
- a dev download the test data once
- there is a new test requiring new data on master
- the dev runs pytest, everything passes because he is lacking new data (or new version of the data)
- tests fail in GitHub CI because the new tests is not actually passing

Note this is not critical, just more convenient for devs.
So it is just a small suggestion for improvement, feel free to merge as it is already.

Reply:
We can do this using functions from ctapipe 0.10 as soon as we updated using `get_dataset_path`.

Reply:
The download script was more a temporal solution until that is there.

Reply:
ha great. then let's merge and we can update later to improve it !